docs/datamodel/indexes.rst

```diff
@@ -184,9 +184,10 @@ Indexes can be augmented with annotations.
   **Foreign and primary keys**
 
   In SQL databases, indexes are commonly used to index *primary keys* and
-  *foreign keys*. In EdgeDB, these fields are automatically indexed; there's no
-  need to manually declare them. Moreover, any property with an
-  :eql:constraint:`exclusive` constraint is also automatically indexed.
+  *foreign keys*. The EdgeDB equivalents of these are the ``id`` property
+  and links; these are automatically indexed and there is no need to manually
```

Comment:
Maybe add "respectively" here just to make it extra clear that primiary key -> EdgeDB `id` and foreign key -> EdgeDB link.

```rst
…are the ``id`` property and links respectively.
```

or even just spell it all out.

```rst
EdgeDB's analog to SQL's primary key is the `id` field that gets automatically
created for each object, and the analog to SQL's foreign key is the link. These
are automatically indexed…
```

Reply:
That spelled-out wording looks nice, have changed to that.